components/style/themes/default.less

```diff
@@ -387,8 +387,8 @@
 @tabs-card-height: 40px;
 @tabs-card-active-color: @primary-color;
 @tabs-title-font-size: @font-size-base;
-@tabs-title-font-size-lg: @font-size-sm;
-@tabs-title-font-size-sm: @font-size-base;
+@tabs-title-font-size-lg: @font-size-lg;
+@tabs-title-font-size-sm: @font-size-sm;
```

Comment:
This one should be `@font-size-base`

Reply:
would it make small tab have 14px font-size? as defined at top:
```
@font-size-base         : 14px;
@font-size-lg           : @font-size-base + 2px;
@font-size-sm           : 12px;
```
I thought small, default and large tab have 12px, 14px and 16px, respectively.

Reply:
Right, the font size of small tabs should be 14px.